.github/ISSUE_TEMPLATE/04-feature-request.yml

```
@@ -0,0 +1,38 @@
+name: Feature Request
+labels: ["status/needs-triage", "type/enhancement"]
+description: Request a feature you'd like in Bottlerocket
+body:
+  - type: markdown
+    attributes:
+      value: |
+        This form is for **feature request** to Bottlerocket. 
+        
+        A few tips before submitting a new issue:
+        - Search for similar features requests, including in [closed issues](https://github.com/bottlerocket-os/bottlerocket/issues?q=is%3Aissue+is%3Aclosed),
+        - Search for similar features already on the [Bottlerocket roadmap](https://github.com/bottlerocket-os/bottlerocket/blob/develop/ROADMAP.md).
+  - type: textarea
+    id: like
+    attributes:
+      label: What I'd like
+      render: bash
+    validations:
+      required: true
+  - type: textarea
+    id: alternatives
+    attributes:
+      label: Any alternatives you've considered
```

for the first person voice, should this be ...

```suggestion
      label: Any alternatives I've considered
```